frontend/src/components/projectEdit/extraIdParams.js

```diff
@@ -0,0 +1,17 @@
+import { styleClasses } from '../../views/projectEdit';
+
+export const ExtraIdParams = ({ value, setProjectInfo }) => {
+  const onInputChange = (e) => {
+    setProjectInfo((p) => ({ ...p, extraIdParams: e.target.value }));
+  };
+
+  return (
+    <input
+      className={styleClasses.inputClass}
+      onChange={onInputChange}
+      type="text"
+      name={`extraIdParams`}
+      value={value}
```

Comment:
let's add a `placeholder` with the same example as the description example: `disabled_features=buildings&offset=-10,5`. Might even pass it in as a param so we can just change it in one spot if we ever need to.